assets/docs/callmemehdi/models/word2vec_swahili/1.md

```diff
@@ -0,0 +1,95 @@
+# Module callmemehdi/word2vec_swahili/1
+
+This is a word2vec model created using Swahili Wikipedia data
+
+<!-- asset-path: https://gsoc-tf.web.app/swahili_word2vec.tar.gz -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
+<!-- license: custom -->
+
+## Overview
+
+We used this [repository](https://github.com/hgrif/wiki-word2vec) to train a word2vec type embeddings on Swahili Wikipedia dumps data.
+We then used these embeddings to create a Keras model that outputs a 100 vectors embedding for each word given.
+
+## Usage
+
+In order to use this model, we need to download the tokenizer
+
+First, download the tokenizer:
+```shell
+curl https://gsoc-tf.web.app/tokenizer.pkl -o tokenizer.pkl
+```
+
+or
+
+```shell
+wget https://gsoc-tf.web.app/tokenizer.pkl
+```
+<br/><br/>
+Then, use it to preprocess your input, here's an example:
+
+```python
+#We load the tokenizer
+import pickle
+
+with open('tokenizer.pkl', 'rb') as handle:
```

Comment:
Can you wrap the tokenizer in a SavedModel as well, so it can be uploaded alongside this one to Hub? That would make for a nicer user experience, eg:

```python
tokenizer = hub.load(...)
model = hub.load(...)

embedding = model(tokenizer(word))
```